### PR TITLE
Sequential page type generator

### PR DESCRIPTION
**Pull request by chillu**
Easier to script, batch delete, etc.
You can run it multiple times and it keeps incrementing the count automatically.
I'll keep the faker dependency around since we'll likely use it for other purposes again.
Also fixed a bug in the system where it was doing the basename() checks in the wrong place.

### CODE REVIEW

File: code/tasks/FTPageTypeCreatorTask.php [hunk 1]
```diff
@@ -41,30 +43,32 @@ public function __construct()
      */
     public function run($request)
     {
-        $count = $request->getVar('count') ?: 20;
-        $module = ModuleLoader::getModule('silverstripe/frameworktest');
-        $testPageDir = $module->getPath() . '/code/test-pages';
+        $totalCount = $request->getVar('count') ?: 20;
```
Comment (maxime-rainville) [inline code/tasks/FTPageTypeCreatorTask.php]: What about making the `GeneratedPageType` customisable as well?

File: code/tasks/FTPageTypeCreatorTask.php [hunk 2]
```diff
@@ -78,17 +82,17 @@ private function getExistingClassNames($dir)
...
-            return $file->getBasename();
+            return basename($file->getBasename(), '.php');
         }, iterator_to_array($files));
     }
 
-    private function generateClassName()
+    private function generateClassName($counter)
     {
-        return ucfirst($this->faker->word . 'Page');
+        return 'GeneratedPageType' . $counter;
```
Comment (maxime-rainville) [inline code/tasks/FTPageTypeCreatorTask.php]: This was the only spot `faker` was used. Can we remove `$this->faker` and `use Faker\Factory;`?

**Comment by maxime-rainville** [comment]
This was blocked because @chillu was on holiday. He's back now. Moving to "Peer Review".

**Comment by maxime-rainville** [comment]
@chillu Are you still working on this? Your last commit says it's a "Work in Progress".

**Comment by chillu** [comment]
Closing as unfinished, might need to pick this up again when we do the GraphQL performance work

**Pull request closed without merging**